File: src/main/java/io/github/fernthedev/secondgame/main/SpriteSheet.java
//
// Source code recreated from a .class file by IntelliJ IDEA
// (powered by Fernflower decompiler)
//

package io.github.fernthedev.secondgame.main;

import java.awt.image.BufferedImage;

public class SpriteSheet {
    private BufferedImage sprite;

    public SpriteSheet(BufferedImage ss) {
        this.sprite = ss;
    }

    public BufferedImage grapImage(int col, int row, int width, int height) {
        BufferedImage img = this.sprite.getSubimage(row * 32 - 32, col * 32 - 32, width, height);
        return img;
    }
}

File: game/src/main/java/io/github/fernthedev/secondgame/main/SpriteSheet.kt
package io.github.fernthedev.secondgame.main

import java.awt.image.BufferedImage

internal class SpriteSheet(private val sprite: BufferedImage) {
    fun grapImage(col: Int, row: Int, width: Int, height: Int): BufferedImage {
        return sprite.getSubimage(row * 32 - 32, col * 32 - 32, width, height)
    }
}